client/js/screens/Profile/ProfileContainer.js

```diff
@@ -8,6 +11,42 @@ export default class ProfileContainer extends Component {
   }
```

Comment:
Remove this contractor if you are not using a state.

Reply:
Done.